manifests/individual_connection.pp

```diff
@@ -0,0 +1,392 @@
+#     will be copied, via ``pki::copy``
+#
+#   * If pki = ``false``, this variable has no effect
+#
+# @param app_pki_dir
+#   The source of certs in the chroot, and the basepath of ``$app_pki_key``,
+#   ``$app_pki_cert``, ``$app_pki_ca``, ``$app_pki_ca_dir``, and
+#   ``$app_pki_crl``
+#
+#   * **NOTE:** Even when using a chroot, stunnel needs the certs to reside
+#     **outside** of the chroot path
+#
+# @param app_pki_key
+#   Path and name of the private SSL key file
+#
+# @param app_pki_cert
+#   Path and name of the public SSL certificate
+#
+# @param app_pki_ca_dir
+#   Since stunnel runs in a chroot, you need to copy the appropriate CA
+#   certificates in from an external source
+#
+#   * This should be the full path to a directory containing **hashed**
+#   versions of the CA certificates
+#
+# @param app_pki_crl
+#   Since stunnel runs in a chroot, you need to copy the appropriate CRL in
+#   from an external source
+#
+# @param fips
+#   Set the ``fips`` global option
+#
+#   * We don't enable FIPS mode by default since we want to be able to use
+#     TLS1.2
+#
+#   * **NOTE:** This has no effect on EL < 7 due to stunnel not accepting the
+#     fips option in that version of stunnel
+#
+# @param connect
+#   Address and port to which to **forward** connections
+#
+#   * For a client, this is the port of the stunnel server
+#   * For the stunnel server, this is the listening port of the tunneled
+#     service
+#   * Just a port indicates that you wish to listen on all interfaces
+#
+#   * Examples:
+#       * ['my.server:3000','my.server2:3001']
+#       * ['my.server:3000']
+#       * ['3000']
+#
+# @param accept
+#   Address and port upon which to **accept** connections
+#
+#   * For a client, this is generally ``localhost``
+#   * For a server, it should be whichever external address is appropriate
+#       * If this is omitted, then connections are accepted on all addresses
+#
+#   * Examples:
+#       * '1.2.3.4:3000'
+#       * '3000'
+#
+# @param client
+#   Indicates that this connection is a client connection
+#
+# @param openssl_cipher_suite
+#   OpenSSL compatible array of ciphers to allow on the system
+#
+# @param ssl_version
+#   Dictate the SSL version that can be used on the system
+#
+#   * This default, combined with the default ``$ciphers``, will only negotiate
+#     at ``TLSv1.1`` or higher
+#
+# @param options
+#   The OpenSSL library options
+#
+#
+# @param trusted_nets
+#   Set this if you don't want to allow all IP addresses to access this
+#   connection
+#
+#   * This only makes sense for servers
+#
+# @param firewall
+#   Include the SIMP ``iptables`` module to manage the firewall
+#
+# @param tcpwrappers
+#   Include the SIMP ``tcpwrappers`` module to manage tcpwrappers
+#
+# All other configuration options can be found in the stunnel man pages
+# @see stunnel.conf(5)
+# @see stunnel.conf(8)
+#
+# @author Nick Markowski <nmarkowski@keywcorp.com>
+#
+define stunnel::individual_connection(
```

Comment:
How about just `stunnel::standalone` or `stunnel::individual`? Connection is redundant.